apps/bare-expo/metro.config.js

```diff
@@ -33,7 +33,44 @@ config.resolver.resolveRequest = (context, moduleName, platform) => {
     const newModuleName = moduleName.replace('react-native', 'react-native-macos');
     return context.resolveRequest(context, newModuleName, platform);
   }
-  return context.resolveRequest(context, moduleName, platform);
+
+  const localModulesModulesPath = path.resolve(__dirname, './.expo/localModules/modules');
+  if (moduleName.endsWith('.module')) {
```

Comment:
you could simplify this if greatly, since the only part that changes is the '.module.js'/'.view.js' string I think